client/src/components/Workbench/Workbench.js

```diff
@@ -39,121 +45,59 @@ function Workbench() {
-              </Alert>
-            </Route>
-            <Route path={`${path}${ROUTES.DATA}`}>
-              <Alert
-                isOpen={welcomeMessages.datafiles}
-                toggle={() => onDismissWelcome('datafiles')}
-                color="secondary"
-                className="welcomeMessage"
-              >
-                This page allows you to upload and manage your files.
-              </Alert>
-            </Route>
-            <Route path={`${path}${ROUTES.APPLICATIONS}`}>
-              <Alert
-                isOpen={welcomeMessages.applications}
-                toggle={() => onDismissWelcome('applications')}
-                color="secondary"
-                className="welcomeMessage"
-              >
-                This page allows you to submit jobs to the HPC systems or access
-                Cloud services using a variety of applications.
-              </Alert>
-            </Route>
-            <Route path={`${path}${ROUTES.ALLOCATIONS}`}>
-              <Alert
-                isOpen={welcomeMessages.allocations}
-                toggle={() => onDismissWelcome('allocations')}
-                color="secondary"
-                className="welcomeMessage"
-              >
-                This page allows you to monitor the status of allocations on the
-                HPC systems and view a breakdown of team usage.
-              </Alert>
-            </Route>
-            <Route path={`${path}${ROUTES.HISTORY}`}>
-              <Alert
-                isOpen={welcomeMessages.history}
-                toggle={() => onDismissWelcome('history')}
-                color="secondary"
-                className="welcomeMessage"
-              >
-                This page allows you to monitor a log of all previous job
-                submissions.
-              </Alert>
-            </Route>
-          </Switch>
-        )}
-        {setupComplete ? (
-          <Switch>
-            <Route path={`${path}${ROUTES.DASHBOARD}`}>
-              <Dashboard />
-            </Route>
-            <Route
-              path={`${path}${ROUTES.ACCOUNT}`}
-              component={ManageAccount}
-            />
-            <Route path={`${path}${ROUTES.DATA}`}>
-              <DataFiles />
-            </Route>
-            <Route
-              path={`${path}${ROUTES.APPLICATIONS}`}
-              component={Applications}
-            />
-            <Route
-              path={`${path}${ROUTES.ALLOCATIONS}`}
-              component={Allocations}
-            />
-            <Route path={`${path}${ROUTES.HISTORY}`} component={History} />
-            <Route
-              path={`${path}${ROUTES.ONBOARDING}`}
-              component={Onboarding}
-            />
-            {showUIPatterns && (
-              <Route path={`${path}${ROUTES.UI}`} component={UIPatterns} />
-            )}
-            <Redirect from={`${path}`} to={`${path}${ROUTES.DASHBOARD}`} />
-          </Switch>
+        {loading ? (
```

Comment:
in addition to waiting for workbench configuration (`workbench.loading`), should we also wait for data files systems here? Or separate ticket for that?

Reply:
Let's tackle that in the data file component if we need to, later

Reply:
👍